src/test/server/backend/repo/test_repo_project.py

```diff
@@ -130,15 +164,6 @@ async def test_empty_model_returns_false(db, pid, user):
     return not await repo.modify(pid, ModifiedProject.construct())
 
 
-@pytest.mark.anyio
-async def test_create_raises(db, user):
-    repo = ProjectRepo(db, *create_config(scopes.ADMIN))
-    with pytest.raises(HTTPException) as e:
-        await repo.create(NewProject(id="adadwdwddaw_Dwdadadawdawd", info=ProjectInfo(name="aaawdad", lang="fi")))
-    assert e.value.status_code == status.HTTP_403_FORBIDDEN
-    assert "Not enough privileges" in e.value.detail
-
-
```

Comment:
Instead of removing this test completely, which drops the coverage, I think this could be changed to test trying to create a project when the scope is `UNAUTHENTICATED`

Reply:
Because in the test_repo_project, the function create_config auto add scopes.AUTHENTICATED, so cannot test with scope UNAUTHENTICATED
![image](https://github.com/MuistotKartalla/muistot-backend/assets/67094879/095c2410-c41f-4b90-a97d-d62bd3757c19)